jsonexample/test/widget_tests_test.dart

```diff
@@ -0,0 +1,193 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jsonexample/dart_convert/converted_simple_object.dart';
+import 'package:jsonexample/dart_convert/converted_complex_object.dart';
+import 'package:jsonexample/widgets.dart';
+
+void main() {
+  group('SimpleObjectView widget test', () {
+    testWidgets('Typical object', (WidgetTester tester) async {
```

Comment:
dtto about test descriptions reading like sentences:

"SimpleObjectView shows JSON contents" or something like that

Reply:
Rewritten.